src/oic/oic/claims_provider.py

```diff
@@ -96,7 +96,7 @@ def _aggregation(self, info):
         logger.info("RESPONSE: %s" % (sanitize(cresp.to_dict()),))
         return cresp
 
-    #noinspection PyUnusedLocal
+    # noinspection PyUnusedLocal
```

Comment:
Removed from `src/*` and `tests/*`. It turned out that those issues are already resolved.